default/scripting/buildings/EXPERIMENTOR_OUTPOST.focs.txt

```diff
@@ -232,12 +232,12 @@ BuildingType
     ]
     icon = ""
 
-// the highest current MaxAIAggression is 5, so this would range from 0.2 to 1.2
+// the highest current MaxAIAggression is 5, so this would range from 1.2 to 2.0
```

Comment:
I think the original was correct...

(0 + 1) / 5 = 0.2
(5 + 1) / 5 = 1.2

Reply:
Ouch. Fixed.